pennylane/ops/op_math/linear_combination.py

```diff
@@ -275,44 +280,45 @@ def wires(self):
     def name(self):
         return "LinearCombination"
 
+    @staticmethod
     @qml.QueuingManager.stop_recording()
-    def _simplify_coeffs_ops(self, cutoff=1.0e-12):
+    def _simplify_coeffs_ops(coeffs, ops, pr, wires, cutoff=1.0e-12):
         """Simplify coeffs and ops
 
         Returns:
             coeffs, ops, pauli_rep"""
 
-        if len(self.ops) == 0:
-            return [], [], self.pauli_rep
+        if len(ops) == 0:
+            return [], [], pr
 
         # try using pauli_rep:
-        if pr := self.pauli_rep:
-
-            wire_order = self.wires
+        if pr is not None:
             if len(pr) == 0:
                 return [], [], pr
 
             # collect coefficients and ops
-            coeffs = []
-            ops = []
+            new_coeffs = []
+            new_ops = []
 
             for pw, coeff in pr.items():
-                pw_op = pw.operation(wire_order=wire_order)
-                ops.append(pw_op)
-                coeffs.append(coeff)
+                pw_op = pw.operation(wire_order=wires)
+                new_ops.append(pw_op)
+                new_coeffs.append(coeff)
 
-            return coeffs, ops, pr
+            return new_coeffs, new_ops, pr
 
-        if len(self.ops) == 1:
-            return self.coeffs, [self.ops[0].simplify()], pr
+        if len(ops) == 1:
+            return coeffs, [ops[0].simplify()], pr
 
-        op_as_sum = qml.sum(*self.operands)
+        op_as_sum = qml.dot(coeffs, ops)
         op_as_sum = op_as_sum.simplify(cutoff)
-        coeffs, ops = op_as_sum.terms()
-        return coeffs, ops, None
+        new_coeffs, new_ops = op_as_sum.terms()
+        return coeffs, ops, pr
```

Comment:
I'm assuming you meant this? Unless I'm missing something.
```suggestion
        return new_coeffs, new_ops, pr
```